wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/templates/xrptimed/Robot.java

```diff
@@ -26,8 +26,7 @@ public class Robot extends TimedRobot {
    * This function is run when the robot is first started up and should be used for any
    * initialization code.
    */
-  @Override
-  public void robotInit() {
+  public Robot() {
```

Comment:
Is the doc comment above still accurate and necessary?

Reply:
Depends on whether we think we need to explain what a constructor is.

Reply:
I think there are plenty of people who are first exposed to programming through FRC and therefore the doc comment is fine to leave